README.md

```diff
@@ -46,262 +46,20 @@ property `db.url` is set to `db:3306`
 
 ## Development
 
-### Pre-requisites
+First read the relevant OS-specific instructions:
 
-##### Dev Container (recommended for Windows)
+- [Windows](getting-started/windows.md)
```

Comment:
Since this information is already contained in the `getting-started/getting-started.md` file, and because the information is more appropriately contained in that file, I recommend removing it entirely from this root `README.md` file.

Subsequent changes to the wording of the next sentence would also be appropriate.